systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -945,6 +946,29 @@ void testReadOnlyRootFileSystem(ExtensionContext extensionContext) {
         ClientUtils.waitForClientsSuccess(testStorage);
     }
 
+    @ParallelNamespaceTest
+    void testDeployUnsupportedKafka(ExtensionContext extensionContext) {
+        final TestStorage testStorage = new TestStorage(extensionContext);
+        String nonExistingVersion = "6.6.6";
+        String nonExistingVersionMessage = "Unsupported Kafka.spec.kafka.version: " + nonExistingVersion + ". Supported versions are:.*";
+        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+
+        resourceManager.createResource(extensionContext, false, KafkaTemplates.kafkaEphemeral(clusterName, 1, 1)
```

Comment:
shouldn't you pass namespaceName here as well?

Reply:
This PR is only for kafkaTopic related changes, i can do another one with other resources.